site/src/components/samplers/modeMedian.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: component
+title: Mode Median Sampling
+component: data/samplers/modeMedian.js
+namespace: samplers
+
+example-code: |
+    var data = fc.data.random.financial()(10000);
+
+    var subsampledData = fc.data.samplers.modeMedian()
```

Comment:
I'd re-write this as:

```
var sampler = fc.data.samplers.modeMedian()
                                         .number(5)
                                         .field('low');

var sampledData = sampler(data);
```

This makes the component construction and use of the component more distinct